index.html

```diff
@@ -0,0 +1,199 @@
+<!DOCTYPE html>
+<html lang="en">
+  <link
+    rel="stylesheet"
+    href="https://stackpath.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css"
+  />
+  <link rel="stylesheet" href="style.css" />
+  <head>
+    <title>StarCraft II: Heart of the Swarm Opening Cinematic</title>
+  </head>
+  <body>
+    <div class="nav">
+      <div>
+        <img
+          class="logo"
+          src="https://www.youtube.com/yts/img/marketing/browsers/yt_logo_rgb_light-vflc4oMnY.png"
+        />
+      </div>
+      <div>
+        <form class="searchbar">
+          <input type="text" placeholder="Search" name="search" />
+          <button type="submit"><i class="fa fa-search"></i></button>
+        </form>
+      </div>
+      <div class="icons">
+        <i class="fa fa-lg fa-th" aria-hidden="true"></i>
+        <i class="fa fa-lg fa-video-camera" aria-hidden="true"></i>
+        <i class="fa fa-lg fa-bell" aria-hidden="true"></i>
+      </div>
+    </div>
+    <div class="main">
+      <div class="video">
+        <iframe
```

Comment:
[REQUIRED] Please use semantic HTML elements instead of just div tags. Use main tag instead of main class. [pathright](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/section/semantic_html_requirement.png)